website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,122 @@
+<link rel="stylesheet" href='/static/css/pages/remove-contributor-page.css'>
+<div id="removeContributor" class="modal fade">
+    <div data-bind="css: modalSize">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+            <div class="modal-body" >
+                <div data-bind="if: canRemoveNode() && !pageChanged()">
```

Comment:
What I like about the comment style of KO control flow is that it works well with syntax highlighters to differentiate control flow from data-binding proper. The [snippet](https://github.com/CenterForOpenScience/osf.io/blob/86ddfa977ec25217002c8401a49042f1e37c0b17/website/templates/project/contributors.mako#L281-L301) screencapped below illustrates both styles. Notice how easy the comments are to parse visually. Now find the `data-bind` control flow in there. Harder, isn't it? ;-)

![screen shot 2016-01-15 at 12 28 15 pm](https://cloud.githubusercontent.com/assets/134455/12360052/9b06ad76-bb83-11e5-94a2-4ede0dcc5255.png)